server/src/main/java/com/ahmadda/domain/organization/OrganizationMember.java

```diff
@@ -88,4 +89,20 @@ public List<Event> getParticipatedEvents() {
     public boolean isAdmin() {
         return this.role == OrganizationMemberRole.ADMIN;
     }
+
+    public void changeRole(final OrganizationMember operator, final OrganizationMemberRole newRole) {
+        validateRoleChangeBy(operator);
+
+        this.role = newRole;
+    }
+
+    private void validateRoleChangeBy(final OrganizationMember operator) {
+        if (!operator.isBelongTo(this.organization)) {
+            throw new ForbiddenException("같은 조직에 속한 조직원만 권한을 변경할 수 있습니다.");
+        }
+
+        if (!operator.isAdmin()) {
+            throw new ForbiddenException("관리자만 조직원의 권한을 변경할 수 있습니다.");
+        }
+    }
```

Comment:
p3
먼가 Admin이 아닌 조직원이 operator가 조직원인지 확인하는게 자연스럽지가 않은데 
```suggestion
    public void changeRole(final List<OrganizationMember> targets, final OrganizationMemberRole newRole) {
        if(!isAdmin()) {
            throw new ForbiddenException("관리자만 조직원의 권한을 변경할 수 있습니다.");
        }
        
        for(OrganizationMember target: targets) {
            if(target.isBelongTo(this.organization) {
                throw new ForbiddenException("같은 조직에 속한 조직원만 권한을 변경할 수 있습니다.");
            }
            target.role = newRole;
        }
    }
```
이런식으로 하는건 어떤가요?